### ISSUE TITLE
Require notification when associating a new authenticator at AAL3

### ISSUE THREAD

**Issue by jimfenton** [usnistgov/800-63-3#1928]
-63B Section 6.1.2.1 says that the CSP SHOULD send a notification to the subscriber when a new authenticator is bound. This should probably be a SHALL at AAL3.

**Comment by jimfenton** [comment]
Possibly even a SHALL at AAL2, but AAL doesn't necessarily establish that we have contact information at all (that's IAL).

**Comment by jerehaynes** [comment]
I agree that it should be a SHALL at even AAL2. I would even go as far to recommend/require a notification be sent to the subscriber [if contact info is associated or available] when the CSP revokes the binding of an authenticator. 
Notifications like these and 'login from new device detected' notifications are helpful in preventing account takeover.
